tests/python/test_bit_operations.py

```diff
@@ -0,0 +1,19 @@
+import taichi as ti
+import numpy as np
+import operator as ops
+from taichi import allclose
+import pytest
+
+func_table = [
+    (ops.lshift, ) * 2,
```

Comment:
Please move the test to `test_element_wise.py`, which test against `m+s`, `s+m`, `m+m`, etc, more systematically.

Reply:
I actually think putting bit operations together is a good idea.

Reply:
I actually think putting things here is better - `test_element_wise` is getting too big and each test there tests too many things. It would be good to have smaller and simpler tests that help to diagnose once anything goes wrong.

For example, `test_bit_operation` should only test bit operations. `test_element_wise` should test the infrastructure of `scalar/matrix`, `matrix/matrix`, `scalar/scalar` (instead of enumerating all possible combinations. We should aim to write **unit** tests.